django/db/migrations/executor.py

```diff
@@ -242,16 +243,23 @@ def apply_migration(self, state, migration, fake=False, fake_initial=False):
                 # Alright, do it normally
                 with self.connection.schema_editor(atomic=migration.atomic) as schema_editor:
                     state = migration.apply(state, schema_editor)
+                    self.record_migrations(migration)
+                    migration_recorded = True
+        if not migration_recorded:
+            self.record_migrations(migration)
+        # Report progress
+        if self.progress_callback:
+            self.progress_callback("apply_success", migration, fake)
+        return state
+
+    def record_migrations(self, migration):
+        """Utility method to record migrations"""
```

Comment:
I think the method name is self documenting and the current docstring doesn't add much value. It should probably be named `record_migration` without an s as well.